stubs/s2clientprotocol/s2clientprotocol/query_pb2.pyi

```diff
@@ -4,19 +4,14 @@ isort:skip_file
 """
 import builtins
 import collections.abc
-import sys
+import typing as typing_extensions
```

Comment:
I can't find a generation script for `s2clientprotocol`. I don't feel too bad about this "manual" change because it's checked by a linter, but still

Reply:
yeah, we don't have an update script for this stubs package... I guess we'll need to manually run mypy-protobuf when the time comes?